### PR TITLE
Make tuples of two Scalars convert to Vectors (using Into)

### PR DESCRIPTION
**Pull request by Xoronic**
## Description

This change is non-breaking, because `Vector::new()` has not been removed.
The update allows API consumers to enter `Vector`s using just a tuple in function arguments.

### Caution!
I suggest merging this PR **after** merging #296. This because #296 is already included in this PR.

## Motivation and Context

This helps with readability and consistency.

**Before**
```rust
Line::newv(Vector::new(50, 80), Vector::new(600, 450)) * Transform::scale((Vector::new(0.5, 0.5)))
```

**After**
```rust
Line::newv((50, 80), (600, 450)) * Transform::scale((0.5, 0.5))
```

## Type of change

New feature (non-breaking change which adds functionality)

## Checks

- [x] I have read `CONTRIBUTING.md`.
- [x] This Pull Request targets the right branch.

**Documentation**
- [x] I have updated `CHANGES.md`, with [BREAKING] next to all breaking changes
- [ ] I have updated the documentation accordingly if necessary

**Testing**
- [x] I have updated / added tests to cover my changes.

**If this Pull Request introduces a breaking change**
- [x] The example found in `README.md` compiles and functions like expected
- [x] The example found in `src/lib.rs` compiles and functions like expected

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,15 +1,14 @@
 # Changelog
 
 ## In-development
-
 - Add new methods of initializing fonts `Font::from_slice` and `Font::from_bytes` using byte sequences
 - [Breaking] Add more fields to `Error`, and add a Result alias to make error bubbling more convenient
-- [Breaking] Renamed type `Draw` to `Sprite` (and updated the readme accordingly)
-- [Breaking] Renamed type `QuicksilverError` to `Error`
-- [Breaking] Added `SaveError`, a new error type
+- [Breaking] Rename type `Draw` to `Sprite` (and updated the readme accordingly)
```
Comment (Xoronic) [inline CHANGES.md]: I also changed these to present-tense

**Comment by Xoronic** [comment]
I forgot to check the wasm compilation, just fixed it.

**Comment by Xoronic** [comment]
It failed without a good reason again..

**Comment by ryanisaacg** [comment]
@Xoronic Rebuilt, hopefully it works.

**Comment by Xoronic** [comment]
Thanks :)

Any way I can do that myself so I won't have to bother you if it happens again?

**Comment by Byter09** [comment]
Kind of off topic, but have you tried the CI provided by GitLab? You don't even need to move the project, just add it as a hook.
Using that for all my projects and it's the best thing ever.

**Comment by ryanisaacg** [comment]
@Xoronic Don't think so, no, unfortunately.

@Byter09 Does it have Windows support? That would be great to not have to have both appveyor and travis.

File: src/graphics/window.rs [hunk 2]
```diff
@@ -261,27 +263,21 @@ impl Window {
     }
 
     ///Handle the available size for the window changing
-    pub(crate) fn adjust_size(&mut self, available: Vector) {
```
Comment (Xoronic) [inline src/graphics/window.rs]: I don't really get this change.. Am I accidently reverting your DPI-fix here? @ryanisaacg

Comment (ryanisaacg) [reply src/graphics/window.rs]: That's what it looks like, yes. Must be a weird git issue because of when this branched off.

Comment (Xoronic) [reply src/graphics/window.rs]: Yep, github says the branch is up to date 😕

**Comment by Byter09** [comment]
@ryanisaacg: I never used any windows related features, but their overview and several search results indicate that yes, indeed they have Windows support. Even PowerShell... I'm at my computer in about 20 min and will look further into it.

**Comment by Xoronic** [comment]
> Kind of off topic, but have you tried the CI provided by GitLab? You don't even need to move the project, just add it as a hook.
Using that for all my projects and it's the best thing ever.

This PR might not be the right place for this conversation :)

**Comment by Xoronic** [comment]
This whole PR became a mess. I am working on a much cleaner and better PR which does the same.

**Comment by ryanisaacg** [comment]
Okay, good luck! Sorry if I was merging too many changes out from under you 😅

**Comment by ryanisaacg** [comment]
Closing this in favor of #312

**Pull request closed without merging**